.evergreen/.evg.yml

```diff
@@ -320,7 +320,7 @@ functions:
         working_dir: "src"
         script: |
           ${PREPARE_SHELL}
-          AUTH=${AUTH} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" TOPOLOGY=${TOPOLOGY} RVM_RUBY="${RVM_RUBY}" .evergreen/run-tests.sh
+          AUTH=${AUTH} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" TOPOLOGY=${TOPOLOGY} RVM_RUBY="${RVM_RUBY}" MONGODB_VERSION=${VERSION} .evergreen/run-tests.sh
```

Comment:
The reason we're setting the version as an environmental variable is due to a quirk in the way that saslprep for SCRAM-SHA-256; because the method `String#unicode_normalize!` wasn't added until Ruby 2.2, we would need to add a dependency to be able to salsprep in older Ruby versions. Rather than add a dependecy for old versions of Ruby to be able to authenticate with SCRAM-SHA-256 in the newest version of the database when SCRAM-SHA-1 still works fine, we decided that it was better to just raise an informative error and wait and see if anyone asks for the feature to be added. Because we create a number of clients in the tests, we need to specify that they use SCRAM-SHA-1 when the Ruby version is below 2.2 and the server version is at least 4.0, so we add the MongoDB server version the tests are running against in order to be able to specify this.

Reply:
Can you please add this and some of the other comments to the code so that they are available to whoever might read the source in the future?